src/modules/vehicle/index.ts

```diff
@@ -123,16 +125,19 @@ export class VehicleModule {
    * @since 5.4.0
    */
   vrm(): string {
-    return `${this.faker.random.alpha({
-      count: 2,
+    return `${this.faker.string.alpha({
+      length: 2,
       casing: 'upper',
     })}${this.faker.datatype.number({
       min: 0,
       max: 9,
     })}${this.faker.datatype.number({
       min: 0,
       max: 9,
-    })}${this.faker.random.alpha({ count: 3, casing: 'upper' })}`.toUpperCase();
+    })}${this.faker.string.alpha({
+      length: 3,
+      casing: 'upper',
+    })}`.toUpperCase();
```

Comment:
```suggestion
    })}${this.faker.string.alpha({
      length: 3,
      casing: 'upper',
    })}`
```
Isn't the `toUpperCase()` call here redundant?